prompt_toolkit/shortcuts/prompt.py

```diff
@@ -1117,7 +1117,7 @@ def _get_continuation(
 
         # When the continuation prompt is not given, choose the same width as
         # the actual prompt.
-        if not continuation and is_true(self.multiline):
+        if not continuation and continuation != '' and is_true(self.multiline):
```

Comment:
This makes sense, but I think I prefer
```python
if continuation is not None and is_true(self.multiline):
```
Because ``None`` is the default here, and for cases where we receive some formatted text, like ``HTML()`` ``ANSI()`` or even ``[]`` (empty list), we want to have an empty continuation. So, not just for an empty string.

Reply:
Sounds good to me. I wasn't sure if anywhere used False or some other falsy value for this so I opted to just check empty string. If that's not the case then checking for None seems like the best solution.